manifest.yml

```diff
@@ -0,0 +1,74 @@
+app-id: com.ilnonop.sonnewebapp
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/run.sh
+finish-args:
+  - --filesystem=home:rw
+  - --socket=x11
+  - --socket=wayland
```

Comment:
it's either or, not both (and wayland has the fallback in that case).

Reply:
corrected!

Reply:
not really. You removed sockets for the UI now.